src/applications/claims-status/utils/helpers.js

```diff
@@ -1110,10 +1111,27 @@ export const buildDateFormatter = (formatString = DATE_FORMATS.LONG_DATE) => {
   };
 };
 
+// Covers two cases:
+//   1. Standard 5103s we get back from the API (only occurs when they're closed).
+//   2. Standard 5103s that we're mocking within our application logic.
+export const isStandard5103Notice = itemDisplayName => {
+  return (
+    itemDisplayName === '5103 Notice Response' ||
+    itemDisplayName === standard5103Item.displayName
+  );
+};
```

Comment:
This could be further split to handle the two different cases, which might be necessary if there are problems with my change to `Default5103EvidenceNotice.jsx`.